src/SSHLibrary/library.py

```diff
@@ -901,13 +901,22 @@ def login(self, username, password, allow_agent=False, look_for_keys=False, dela
         | `Open Connection` | linux.server.com | prompt=$         |
         | ${output}=        | `Login`          | johndoe          | secretpasswd |
         | `Should Contain`  | ${output}        | johndoe@linux:~$ |
+
+        Example that logs in through a proxy server (10.128.3.101)
+        | `Open Connection` | linux.server.com |
+        | ${output}=        | `Login`          | johndoe       | secretpasswd |
+/usr/bin/ssh -l robot -i ~/.ssh/robot_id_rsa -o StrictHostKeyChecking=no -o
+UserKnownHostsFile=/dev/null -W %h:%p 10.128.3.101 |
+        | `Should Contain`  | ${output}        | Last login at |
+        | 
```

Comment:
The table formating is off, removing the `|` from this line might fix it. You can check by generating keyword documentation with `robot.libdoc`.